app/helpers/pom_helper.rb

```diff
@@ -36,9 +36,8 @@ def grade(pom)
   end
 
   def fetch_pom_name(staff_id)
-    pom_firstname, pom_secondname =
-      PrisonOffenderManagerService.get_pom_name(staff_id)
-    "#{pom_secondname}, #{pom_firstname}"
+    staff = HmppsApi::PrisonApi::PrisonOffenderManagerApi.staff_detail(staff_id)
+    "#{staff.last_name}, #{staff.first_name}"
```

Comment:
I'm not convinced our View layer should be calling out to our API... 😕
But I realise you haven't introduced that behaviour in this PR. It was always there, just hidden in the Service object.

Reply:
Thanks for the comment - turns out it isn't. Moved this 'helper' method in the other direction into the service